arrowhead-clients/consumer.py

```diff
@@ -0,0 +1,37 @@
+import time
+import urllib3
+import sys
+sys.path.insert(1, '/home/ruben/git/client-library-python')
+import arrowhead_client.api as ar
```

Comment:
The provider and consumer will be using this library:
https://github.com/arrowhead-f/client-library-python

Reply:
Alright. I'll make sure it's available on the RPi. In the meantime, use it as a regular library instead of modifying `PATH`. That is, `import arrowhead-client` (or however it exposes itself) instead.

Reply:
The library is not needed in the RPi because the arrowhead node is changed (lcm-arrowhead.py). It is only needed where consumer and provider script is running, they will be running on a server.

Reply:
Then the library will be installed on the server. But import it instead; `/home/ruben/` doesn't exist on the server either.

Reply:
The linked library has been packaged into the `src/shell.nix` expression. Use it with `import arrowhead_client`.